### ISSUE TITLE
Can not running well on Jetson Xavier NX

### ISSUE THREAD

**Issue by Alexleisandao** [CMU-Perceptual-Computing-Lab/openpose#2123]
Issue Summary
After compiling without any error Openpose 1.7 on Xavier NX (Jetpack 4.5), running:
./build/examples/openpose/openpose.bin --video ./examples/media/video.avi --net_resolution "256x144" makes it freeze for minutes, before the process gets killed.

There is no issue with a smaller net-resolution like "128x96" (at least no crash, but it takes up to 3 min to run!), but I am surprised because I was using this net-resolution on the Jetson TX2 and Openpose 1.6 (and even bigger net-resolution) and everything was working.

I can see with jtop that the memory (8GB) gets full pretty quickly.

Type of Issue
Execution error
Your System Configuration
Whole console output
./build/examples/openpose/openpose.bin --video ./examples/media/video.avi --net_resolution "256x144"
Starting OpenPose demo...
Configuring OpenPose...
Starting thread(s)...
Auto-detecting all available GPUs... Detected 1 GPU(s), using 1 of them starting at GPU 0.
killed
OpenPose version: 1.7

General configuration:

Installation mode:
cmake .. -D CUDA_HOST_COMPILER=/usr/bin/cc \
   -D CUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda \
   -D CUDA_USE_STATIC_CUDA_RUNTIME=ON \
   -D CUDA_rt_LIBRARY=/usr/lib/aarch64-linux-gnu/librt.so \
   -D CUDA_ARCH_BIN=7.2 \
   -D GPU_MODE=CUDA \
   -D DOWNLOAD_FACE_MODEL=ON \
   -D DOWNLOAD_COCO_MODEL=ON \
   -D USE_OPENCV=ON \
   -D BUILD_PYTHON=OFF \
   -D BUILD_EXAMPLES=ON \
   -D BUILD_DOCS=OFF \
   -D DOWNLOAD_HAND_MODEL=ON
   make -j`nproc`
Operating system: Jetpack 4.5
cat /etc/nv_tegra_release 
 R32 (release), REVISION: 5.1, GCID: 26202423, BOARD: t186ref, EABI: aarch64
3rd-party software:

Caffe version: Default from OpenPose
CMake version: 3.23.0
GPU mode issue:

CUDA version: 10.2
cuDNN version: 8.0

**Comment by Alexleisandao** [comment]
> 
Anyone else have fixed it? Please help me.It is been bothering me for a long time.

**Comment by xiepip** [comment]
I came across the same issue.
Did you fix it? Thanks!@Alexleisandao

**Comment by Alexleisandao** [comment]
您的邮件我已经收到，谢谢！

**Comment by Alexleisandao** [comment]
I did not fix it,but I finish install light-weight-openpose successfully.Although light-weight-openpose can be run, but it only can meet about 2 fps. So,it can not meet my subject's requirements.&nbsp;
If you fix it,please tell me ,Thank you very much.
Do you have WeChat? We can communicate by it!




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月26日(星期五) 晚上8:56
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [CMU-Perceptual-Computing-Lab/openpose] Can not running well on Jetson Xavier NX (Issue #2123)





 
I came across the same issue.
 Did you fix it? ***@***.***

**Comment by 274112211** [comment]
I have fixed it. Through install CUDA10.0 and CUDNN7.3. So，I think the problem is the CUDA10.2 and CUDNN8 version before does not support.

**Comment by Alexleisandao** [comment]
您的邮件我已经收到，谢谢！

**Comment by fridayhoho** [comment]
> I have fixed it. Through install CUDA10.0 and CUDNN7.3. So，I think the problem is the CUDA10.2 and CUDNN8 version before does not support.

downgrade the CUDA version?  from CUDA10.2 to CUDA10?

**Comment by EndlessLoops** [comment]
> > I have fixed it. Through install CUDA10.0 and CUDNN7.3. So，I think the problem is the CUDA10.2 and CUDNN8 version before does not support.
> 
> downgrade the CUDA version? from CUDA10.2 to CUDA10?

Are you fix it ？

**Comment by Alexleisandao** [comment]
您的邮件我已经收到，谢谢！

**Comment by Alexleisandao** [comment]
yes,&nbsp;I have fixed it on jetson.you can try it by my method!
But, I meet similar problem on another computer with Video card A2000. But I have no time to try to fix it on that computer, do you have idea to help me?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年5月24日(星期三) 上午10:54
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [CMU-Perceptual-Computing-Lab/openpose] Can not running well on Jetson Xavier NX (Issue #2123)





   
I have fixed it. Through install CUDA10.0 and CUDNN7.3. So，I think the problem is the CUDA10.2 and CUDNN8 version before does not support.
  
downgrade the CUDA version? from CUDA10.2 to CUDA10?
  
Are you fix it ？
